src/core/Akka.FSharp.Tests/app.config

```diff
@@ -3,6 +3,14 @@
   <configSections>
     <section name="akka" type="Akka.Configuration.Hocon.AkkaConfigurationSection, Akka" />
   </configSections>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="FSharp.Core" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-4.3.1.0" newVersion="4.4.2.0" />
```

Comment:
Help wanted: I'm not able to make R# test runner to pick up dependency redirects for FSharp.Core.